sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/Models/MountFileShareConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Workloads.Models
+{
+    /// <summary> Gets or sets the file share configuration for externally mounted cases. </summary>
+    public partial class MountFileShareConfiguration : FileShareConfiguration
+    {
+        /// <summary> Initializes a new instance of MountFileShareConfiguration. </summary>
+        /// <param name="id"> The fileshare resource ID. </param>
+        /// <param name="privateEndpointId"> The private endpoint resource ID. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="id"/> or <paramref name="privateEndpointId"/> is null. </exception>
+        public MountFileShareConfiguration(string id, string privateEndpointId)
+        {
+            Argument.AssertNotNull(id, nameof(id));
+            Argument.AssertNotNull(privateEndpointId, nameof(privateEndpointId));
+
+            Id = id;
+            PrivateEndpointId = privateEndpointId;
+            ConfigurationType = ConfigurationType.Mount;
+        }
+
+        /// <summary> Initializes a new instance of MountFileShareConfiguration. </summary>
+        /// <param name="configurationType"> The type of file share config. </param>
+        /// <param name="id"> The fileshare resource ID. </param>
+        /// <param name="privateEndpointId"> The private endpoint resource ID. </param>
+        internal MountFileShareConfiguration(ConfigurationType configurationType, string id, string privateEndpointId) : base(configurationType)
+        {
+            Id = id;
+            PrivateEndpointId = privateEndpointId;
+            ConfigurationType = configurationType;
+        }
+
+        /// <summary> The fileshare resource ID. </summary>
+        public string Id { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
Kept it string, same as swagger: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/resource-manager/Microsoft.Workloads/preview/2022-11-01-preview/SAPVirtualInstance.json#L2995